src/Audacia.Core/Extensions/ExpressionEnumerableExtensions.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+
+namespace Audacia.Core.Extensions
+{
+    /// <summary>
+    /// Expression for an Enumerable of <see cref="Expression"/>s.
+    /// </summary>
+    public static class ExpressionEnumerableExtensions
+    {
+        /// <summary>
+        /// Reduce multiple expressions with AND logic into a single expression.
+        /// </summary>
+        /// <param name="expressions">The list of expressions to combine.</param>
+        /// <typeparam name="T">The type of the source of the predicate.</typeparam>
+        /// <returns>A combined expression representing the OR logic of all input expressions.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="expressions"/> is null.</exception>
+        public static Expression<Func<T, bool>> Any<T>(
+            this List<Expression<Func<T, bool>>> expressions)
```

Comment:
🔧 Does this want to be modify to` IEnumerable<T>` instead of `List<T>`

Reply:
good point - done. 

I've also simplified it a lot with `Aggregate`

Reply:
Nice 👍